data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -137,11 +112,11 @@ def test_get_output_file_path(self, mock_etl, mock_paths):
         )
         assert output_file_path == expected_output_file_path
 
-    def test_extract(self, mock_etl, mock_paths):
-        """Ensure the extract results are working as appropriate."""
+    def test_extract_path(self, mock_etl, mock_paths):
+        """Ensure the extract is happening where appropriate."""
+        # this is returning true for no reason?
         tmp_path = mock_paths[1]
 
-        # ETL instance returned by this method is never used, so we assign it to `_`.
```

Comment:
I found this comment kind of helpful, was there a reason we wanted to take it out?

Reply:
added it back!